### ISSUE TITLE
Bundler resolve looking in the wrong place

### ISSUE THREAD

**Issue by marklam** [johtela/litscript#1]
Is it possible to override the paths that the bundler uses?
I'm having a problem where during the resolve for main.ts, it looks for ts-loader in paths like:
`….. \node_modules\litscript\node_modules\ts-loader`

(litscript is installed in a local node_modules, which may have something to do with it)

**Comment by johtela** [comment]
I could reproduce the problem, and it happens indeed when litscript is installed locally. To find the root cause I have to play around with the Webpack configuration. As a workaround installing litscript globally should work ok.

**Comment by marklam** [comment]
Thanks for looking into it. 
Unfortunately I can't install npm tools globally as part of the CI build script, but as a workaround I'm copy any `*-loader` directories from `node_modules` to `node_modules\litscript\node_modules` (I don't have permission to create symlinks either).

**Comment by johtela** [comment]
The bug should be now fixed. Problem was that I had set the `node_modules` path for loaders in the Webpack configuration to point to a directory relative to the `bundler.js` module: `../node_modules` But the directory is different when npm installs LiTScript locally. In that case `litscript` and `ts-loader` directory are siblings.

So, I added a function that checks where the `node_modules` directory is and sets it correctly. I verified that the fix worked for me. Reopen the issue, if you still have problems.
